unable to install units on Debian Jessie

Hi,

tried to install R package units with this command:

root@rserver:~# su - -c "R -e \"install.packages('units', repos='http://cran.rstudio.com/')\""

R version 3.4.3 (2017-11-30) -- "Kite-Eating Tree"
Copyright (C) 2017 The R Foundation for Statistical Computing
Platform: x86_64-pc-linux-gnu (64-bit)

R ist freie Software und kommt OHNE JEGLICHE GARANTIE.
Sie sind eingeladen, es unter bestimmten Bedingungen weiter zu verbreiten.
Tippen Sie 'license()' or 'licence()' fÃ¼r Details dazu.

R ist ein Gemeinschaftsprojekt mit vielen Beitragenden.
Tippen Sie 'contributors()' fÃ¼r mehr Information und 'citation()',
um zu erfahren, wie R oder R packages in Publikationen zitiert werden kÃ¶nnen.

Tippen Sie 'demo()' fÃ¼r einige Demos, 'help()' fÃ¼r on-line Hilfe, oder
'help.start()' fÃ¼r eine HTML Browserschnittstelle zur Hilfe.
Tippen Sie 'q()', um R zu verlassen.

> install.packages('units', repos='http://cran.rstudio.com/')
Installiere Paket nach â/usr/local/lib/R/site-libraryâ
(da âlibâ nicht spezifiziert)
versuche URL 'http://cran.rstudio.com/src/contrib/units_0.6-0.tar.gz'
Content type 'application/x-gzip' length 912393 bytes (891 KB)
==================================================
downloaded 891 KB

* installing *source* package âunitsâ ...
** Paket âunitsâ erfolgreich entpackt und MD5 Summen Ã¼berprÃ¼ft
configure: units: 0.6-0
checking whether the C++ compiler works... yes
checking for C++ compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking how to run the C++ preprocessor... g++ -E
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for stdbool.h that conforms to C99... yes
checking for _Bool... no
checking for error_at_line... yes
checking for gcc... gcc -std=gnu99
checking whether we are using the GNU C compiler... yes
checking whether gcc -std=gnu99 accepts -g... yes
checking for gcc -std=gnu99 option to accept ISO C89... none needed
checking for XML_ParserCreate in -lexpat... no
checking udunits2.h usability... yes
checking udunits2.h presence... yes
checking for udunits2.h... yes
checking for ut_read_xml in -ludunits2... yes
configure: creating ./config.status
config.status: creating src/Makevars
** libs
g++  -I/usr/share/R/include -DNDEBUG  -I"/usr/local/lib/R/site-library/Rcpp/include"    -fpic  -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -D_FORTIFY_SOURCE=2 -g  -c RcppExports.cpp -o RcppExports.o
In file included from RcppExports.cpp:4:0:
../inst/include/units.h:7:53: error: wrong number of template arguments (4, should be 3)
 typedef XPtr<ut_unit,PreserveStorage,finalizeUT,true> XPtrUT;
                                                     ^
In file included from /usr/local/lib/R/site-library/Rcpp/include/Rcpp.h:46:0,
                 from ../inst/include/units.h:2,
                 from RcppExports.cpp:4:
/usr/local/lib/R/site-library/Rcpp/include/Rcpp/XPtr.h:48:7: error: provided for âtemplate<class T, template<class> class StoragePolicy, void (* Finalizer)(T*)> class Rcpp::XPtrâ
 class XPtr :
       ^
In file included from RcppExports.cpp:4:0:
../inst/include/units.h:7:61: error: invalid type in declaration before â;â token
 typedef XPtr<ut_unit,PreserveStorage,finalizeUT,true> XPtrUT;
                                                             ^
/usr/lib/R/etc/Makeconf:168: recipe for target 'RcppExports.o' failed
make: *** [RcppExports.o] Error 1
ERROR: compilation failed for package âunitsâ
* removing â/usr/local/lib/R/site-library/unitsâ

Die heruntergeladenen Quellpakete sind in
        â/tmp/Rtmp0OcR6R/downloaded_packagesâ
Warnmeldung:
In install.packages("units", repos = "http://cran.rstudio.com/") :
  Installation des Pakets âunitsâ hatte Exit-Status ungleich 0

required libudunits2-dev is the latest version

root@rserver:~# sudo apt-get install libudunits2-dev
Paketlisten werden gelesen... Fertig
AbhÃ¤ngigkeitsbaum wird aufgebaut.
Statusinformationen werden eingelesen.... Fertig
libudunits2-dev ist schon die neueste Version.
0 aktualisiert, 0 neu installiert, 0 zu entfernen und 137 nicht aktualisiert.

Need to install the R Package r-spatial.github.io/sf/ but it depends on units package that I cannot install. What can I do ?

thanks in advance
regards

--- a/DESCRIPTION
+++ b/DESCRIPTION
@@ -12,7 +12,7 @@ Depends:
 Imports:
 	Rcpp
 LinkingTo:
-	Rcpp
+	Rcpp (>= 0.12.10)
 Suggests:
 	udunits2,
     NISTunits,

